client/pokemon/pokemon.ctrl.js

```diff
@@ -329,4 +329,4 @@ function parseDate(timestamp) {
 function replace3dsUnicodeChars(str) {
   return str &&
     str.replace(/[\ue08e-\ue09d]/g, c => '♂♀♠♣♥♦★◎○□△◇♪☀☁☂'[c.charCodeAt(0) - 0xe08e] + '\ufe0e');
-}
```

Comment:
I think this'll break the linter because you removed the empty line at the end of the file.